src/pages/home/home.ts

```diff
@@ -1,133 +1,233 @@
-import { Component } from '@angular/core/';
+import { Component } from "@angular/core/";
 import {
   GoogleMaps,
   GoogleMap,
   GoogleMapsEvent,
-  GoogleMapOptions,
-  CameraPosition,
-  MarkerOptions,
-  Marker,
-} from '@ionic-native/google-maps';
-import { Platform } from 'ionic-angular';
-import { YelpService } from '../../app/services/yelp/yelp.service';
-import { Observable } from 'rxjs/Observable';
-import { Geolocation } from '@ionic-native/geolocation';
-import { WorkfromService } from '../../app/services/workfrom/workfrom.service';
+  GoogleMapOptions
+} from "@ionic-native/google-maps";
+import { Platform, ModalController, NavParams } from "ionic-angular";
+import { WorkfromService } from "../../app/services/workfrom/workfrom.service";
+import { SpacePage } from "../space/space";
+import { GroupSocketService } from "../../app/services/groupsocket/groupsocket.service";
+import geolib from "geolib";
+import {UserInfo} from "../../app/services/groupsocket/groupsocket.service";
+
+interface Coordinates {
+  latitude: number;
+  longitude: number;
+}
+
+const RANDOM_GEOCOORDINATES: Coordinates[] = [
+  { latitude: 25.992046, longitude: -80.283645 }, // Pembroke Pines
+  { latitude: 25.942871, longitude: -80.12338 } // Sunny Isles
+  // { latitude: 38.5678818, longitude: -121.4636956 }, // East Sacramento
+  // { latitude: 37.2972316, longitude: -122.0976092 }, // San Jose
+];
 
 @Component({
-  selector: 'page-home',
-  templateUrl: 'home.html',
+  selector: "page-home",
+  templateUrl: "home.html"
```

Comment:
Oh no! I see that the single quotes here have been changed to double quotes. I am assuming that's the formatter you have. Well, for javascript code, it should be single quotes. Please change it back if you can.

Reply:
Changed back to single quotes.